### PR TITLE
feature/add admin login button

### LINKED ISSUE #37: Add 'admin login' button
Button should be accessible on every page of application, in nav bar (see CRA example)

Image

### PR DESCRIPTION
**Pull request by Justinzoorob**

### CODE REVIEW

**Review by mputnik — APPROVED** [review]

**Review by khalidkanaan — APPROVED** [review]

**Review by jjsoong — APPROVED** [review]

**Comment by hc-rabid** [comment]
Please await my approval before merging feature branches next time around

**Pull request merged**
